packages/Checkbox/src/Checkbox.spec.tsx

```diff
@@ -70,9 +87,18 @@ describe('packages/Checkbox', () => {
         onChange={uncontrolledOnChange}
       />,
     ).container.firstChild;
+
+    if (!isElement(uncontrolledContainer)) {
+      throw new Error('Could not find uncontrolled container component');
+    }
+
     const uncontrolledCheckbox = uncontrolledContainer.children[0];
 
-    fireEvent.click(uncontrolledContainer.firstChild);
+    if (!isInput(uncontrolledCheckbox)) {
+      throw new Error('Could not find checkbox input element');
+    }
+
+    fireEvent.click(uncontrolledCheckbox);
```

Comment:
Could someone confirm for me that this is doing the right thing? `.firstChild` should be the same as `.children[0]` afaik, but the tests say "when the label is clicked", not "when the checkbox is clicked" so I wanna make sure I'm not missing something.